mapboxgl/viz.py

```diff
@@ -345,3 +360,88 @@ def add_unique_template_variables(self, options):
             lineWidth=self.line_width,
         ))
 
+
+class AltChoroplethViz(MapViz):
+    """Create a choropleth viz"""
+
+    def __init__(self,
+                 data,
+                 vector_polygon_source=False,
```

Comment:
Could we automatically sniff this based on if a specific class variable is not `None`, such as `vector_url`?

Reply:
Sure thing.

Reply:
Took care of this in b11c006.